src/components/Label/Label.scss

```diff
@@ -189,7 +192,8 @@ $transition-timing-function: ease-in-out;
             --_--bg-color-hover: var(--g-color-base-simple-hover);
             --_--text-color: var(--g-color-text-complementary);
 
-            box-shadow: inset 0 0 0 1px var(--g-color-line-generic);
+            box-shadow: inset 0 0 0 var(--g-label-border-width, 1px)
```

Comment:
In order to have `--g-label-border-width` variable, we should define private variable for that at first and use it like other variables in every theme. Same applies for `--g-label-border-color`